gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -78,6 +79,7 @@ def step(self, action, verbose=True):
         if not self.action_space.contains(action):
             raise error.InvalidAction(f'Action must be an integer between '
                                       f'0 and {self.action_space.n}!')
+        prev_state = deepcopy(self.event)
```

Comment:
You're copying something that might be in megabytes in size just to compare if inventory has increased? Why not just deepcopy the inventory or even just keep track of the len of the list as I did? Why copy whole event?